components/brave_wallet_ui/components/extension/confirm-transaction-panel/confirm-transaction-panel.tsx

```diff
@@ -222,23 +240,36 @@ export const ConfirmTransactionPanel = ({
               eTldPlusOne={originInfo.eTldPlusOne}
             />
           </URLText>
-          <FromToRow>
-            <Tooltip
-              text={fromAddress}
-              isAddress={true}
-              position='left'
-            >
-              <AccountNameText>{fromAccountName}</AccountNameText>
-            </Tooltip>
+          <Row marginBottom={8} maxWidth={isContract ? '90%' : 'unset'} width='unset'>
+            <Row maxWidth={isContract ? '70px' : 'unset'} width='unset'>
+              <Tooltip
+                text={fromAddress}
+                isAddress={true}
+                position='left'
+              >
+                <AccountNameText>{fromAccountName}</AccountNameText>
+              </Tooltip>
+            </Row>
             <ArrowIcon />
-            <Tooltip
-              text={transactionDetails.recipient}
-              isAddress={true}
-              position='right'
-            >
-              <AccountNameText>{reduceAddress(transactionDetails.recipient)}</AccountNameText>
-            </Tooltip>
-          </FromToRow>
+            {isContract ? (
+              <Column alignItems='flex-start' justifyContent='flex-start'>
+                <NetworkText>
+                  {getLocale('braveWalletNFTDetailContractAddress')}
+                </NetworkText>
+                <ContractButton onClick={onClickViewOnBlockExplorer('contract', `${transactionDetails.recipient}`)}>
```

Comment:
this could call `window.open` in components/brave_wallet_ui/utils/block-explorer-utils.ts - could you make sure that sets `noopener`, ex: `window.open('https://example.com', '_blank', 'noopener')`

this is now the default for anchor elements, but i'm not sure if it applies to window.open. https://chromestatus.com/feature/6140064063029248

Reply:
Pushed an update :) https://github.com/brave/brave-core/compare/56d656d4b4be1464fc805543e7bf28e47c3e2998..05c6fda7f0ccc5b763e99d5e22f432b7050e1efe